packages/botonic-cli/src/commands/new.ts

```diff
@@ -95,9 +100,16 @@ Creating...
       spinner: 'bouncingBar',
     }).start()
     const dependencyCommand = `npm install`
-    await exec(dependencyCommand)
+    try {
+      await exec(dependencyCommand)
+    } catch (e) {
+      const error = `Installing dependencies: ${String(e)}`
+      trackError('Created Botonic Bot CLI', {
```

Comment:
since we always want an error arg for trackError, I prefere that it has 3 arguments: eventName, error, and optional properties